streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -360,7 +360,8 @@ public void suspend() {
      *   - commit offsets
      * </pre>
      */
-    private void suspend(final boolean clean) {
+    // visible for testing
+    void suspendImpl(final boolean clean) {
```

Comment:
The names are terrible. You don't suspendImpl something, you suspend it. You don't commitImpl, you commit.